crowbar_framework/config/experimental.yml

```diff
@@ -0,0 +1,14 @@
+default: &default
+  skip_unready_nodes:
+    enabled: false
+    roles:
+     - dns-client
```

Comment:
Btw, here's a list of roles I would add here:

- deployer-client
- provisioner-base
- ipmi
- bmc-nat-client
- dns-client
- logging-client
- ntp-client
- suse-manager-client
- updater
- nova-compute-ironic
- nova-compute-kvm
- nova-compute-qemu
- nova-compute-vmware
- nova-compute-xen
- nova-compute-zvm
- ceilometer-agent